launcher/InstanceCreationTask.cpp

```diff
@@ -31,7 +31,7 @@ void InstanceCreationTask::executeTask()
         components->buildingFromScratch();
         components->setComponentVersion("net.minecraft", m_version->descriptor(), true);
         if(m_usingLoader)
-            components->setComponentVersion(m_loader, m_loaderVersion->descriptor(), true);
+            components->setComponentVersion(m_loader, m_loaderVersion->descriptor());
```

Comment:
Yeah, I wasn't sure what that did. The dangers of copying and pasting, I suppose lol

Reply:
Yeah no problem. We didn't notice in the initial review either :D